en/02_Developer_Guides/10_Email/index.md

```diff
@@ -6,101 +6,78 @@ icon: envelope-open
 
 ## Configuration
 
-Silverstripe CMS provides an API over the top of the [SwiftMailer](http://swiftmailer.org/) PHP library which comes with an extensive list of "transports" for sending mail via different services. 
+Silverstripe CMS provides an API over the top of [symfony/mailer](https://symfony.com/doc/current/mailer.html) which comes with an extensive list of "transports" for sending mail via different services such as SMTP, Gmail and Amazon SES.
 
-For legacy reasons, Silverstripe CMS defaults to using the built-in PHP `mail()` command via a deprecated class `Swift_MailTransport`. However, using this layer is less secure and is strongly discouraged.
+Email configuration is done using Symfony's `[DSN](https://symfony.com/doc/current/mailer.html#transport-setup) configuration string which is used to select which transport is used and any required configuration such as username and password. In Silverstripe, this is done with either an environment variable or yml configuration.
 
-It's highly recommended you upgrade to a more robust transport for additional security. The Sendmail transport is the most common one. The `sendmail` binary is widely available across most Linux/Unix servers.
+The `Sendmail` transport is the most common one and is used by default in Silverstripe. The `sendmail` binary is widely available across most Linux/Unix servers. By default the sendmail command used is `/usr/sbin/sendmail -bs`.
 
-You can use any of the Transport classes provided natively by SwiftMailer. There are also countless PHP libraries offering custom Transports to integrate with third party mailing service:
-- read the [SwiftMailer Transport Types documentation](https://swiftmailer.symfony.com/docs/sending.html#transport-types) for a full list of native Transport
-- search [Packagist for SwiftMailer Transport](https://packagist.org/?query=SwiftMailer+Transport) to discover additional third party integrations
+Alternatively you can provide a different `DSN` to select any of the Transport classes provided natively by `symfony/mailer` or other compatible third-party transports. For more information and to see what other transports are available see the [symfony/mailer transport types](https://symfony.com/doc/current/mailer.html#using-a-3rd-party-transport).
 
-To swap out the transport used by the `Mailer`, create a file `app/_config/email.yml`
+To set the DSN string in an environment variable (recommended):
 
-To use a `sendmail` binary:
-
-```yml
----
-Name: myemailconfig
-After:
-  - '#emailconfig'
----
-SilverStripe\Core\Injector\Injector:
-  Swift_Transport:
-    class: Swift_SendmailTransport
+**.env**
+```
+MAILER_DSN="<my-dsn>"
 ```
 
-To use SMTP:
+To set the DSN string in project yml:
 
+**app/_config/myemail.yml**
 ```yml
 ---
-Name: myemailconfig
-After:
-  - '#emailconfig'
+Name: mailer-dsn-config
+After: '*'
```

Comment:
I think it's safest to. This means it will always have precedence.

Technically it could possibly be `After: 'mailer-dsn-env'` - relates to https://github.com/silverstripe/silverstripe-framework/pull/10494/files

However
- I'm not 100% sure if 'mailer-dsn-env' yml block will always exist, as there's an `only` conditional in there which checks to see the `MAILER_DSN` env variable exists.
- we might need to add in another yml block that goes after 'mailer-dsn-env' at a later date, which would then take precedence over project config

There's a different argument to be made that the env variable should have precedence over any project yml.

Probably makes sense to leave that argument for the actual framwork PR though

Should leave this unresolved for now

Reply:
I agree that this should be changed so that the .env config takes precedence (see https://github.com/silverstripe/silverstripe-framework/pull/10494/files#r985306124). Please update the docs to match.

Reply:
Have updated docs, note: I've have kept the `MAILER_DSN` config in yml, rather than overriding in php.  This is because the service in created via new injector factory config which is all set in yml.  It also allows more flexibility in the case where there's a misconfigured env variable on a prod server that needs to be urgently changed by developers who can't quickly change an env variable (though I haven't mentioned that explicitly in the docs) I've updated the docs accordingly, specifically mentioning that project config must be exactly naming `mailer-dsn-project-config` to allow the env variable to retain precedence.